src/components/ProfileTree/ProfileBranch.js

```diff
@@ -113,14 +130,18 @@ export const ProfileBranch = memo(({
                 matchData.map((item, i) => (
                   <ProfileBranch
                     key={`profile-branch-${item.id}-${i}`}
-                    entityKey={`${camelCase(item.contentType)}s`}
-                    recordData={item.content}
-                    contentData={item.childSnapshotWrappers}
-                    record={record}
+                    reactTo="MATCH"
                     linkingRules={linkingRules}
+                    recordData={item}
```

Comment:
`record` and `recordData` are two very similar nouns for me. Please differentiate them more if possible.